pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -112,13 +112,14 @@ def _update_actions_len(self):
             self.actions_len = ListOfActions(self.actions).get_size()
 
     def _validate_in_port(self):
-        port = self.in_port
-        valid = True
-        if isinstance(port, Port):
-            if port not in _VIRT_IN_PORTS:
-                valid = False
-        elif isinstance(port, int) and (port < 1 or port >=
-                                        Port.OFPP_MAX.value):
-            valid = False
-        if not valid:
-            raise ValidationError('{} is not a valid input port.'.format(port))
+        """Validate in_port attribute.
```

Comment:
@macartur Please, add a line here commenting the validation criteria. What is an `invalid port`?